microcosm_postgres/tests/test_company_store.py

```diff
@@ -203,3 +208,106 @@ def test_delete_company_with_employees(self):
             ).create()
 
         assert_that(calling(company.delete), raises(ReferencedModelError))
+
+    def test_create_company_stores_metrics(self):
+        with patch.object(self.graph.company_store, "postgres_store_metrics") as mocked_metrics:
+            with transaction():
+                company = Company(
+                    name="name",
+                    type=CompanyType.private,
+                ).create()
+
+            retrieved_company = Company.retrieve(company.id)
+            assert_that(retrieved_company.name, is_(equal_to("name")))
+            assert_that(retrieved_company.type, is_(equal_to(CompanyType.private)))
+
+            assert_that(mocked_metrics.call_count, is_(equal_to(2)))
+            call_arg1 = mocked_metrics.call_args_list[0][1]
+            call_arg2 = mocked_metrics.call_args_list[1][1]
+            assert_that(call_arg1, has_entries(dict(execution_result=equal_to(SQLExecutionStatus.SUCCESS),
```

Comment:
I'm going to be a bit annoying about style:

```suggestion
            assert_that(call_arg1, has_entries(dict(
                execution_result=equal_to(SQLExecutionStatus.SUCCESS),
                model_name=equal_to('Company'),
                action=equal_to('create'),
            )))
```
You might be asking yourself: wouldn't it be easier to have auto-formatting?

...

Yes. Yes it would.

Reply:
Done